### PR TITLE
Fix closed tickets stats count

### PR DESCRIPTION
**Pull request by protich**
Addresses issue #525

### CODE REVIEW

File: include/class.ticket.php [hunk 1]
```diff
@@ -1764,8 +1764,7 @@ function getStaffStats($staff) {
                         AND assigned.staff_id='.db_input($staff->getId()).')'
             .' LEFT JOIN '.TICKET_TABLE.' closed
                 ON (closed.ticket_id=ticket.ticket_id
-                        AND closed.status=\'closed\'
-                        AND closed.staff_id='.db_input($staff->getId()).')'
+                        AND closed.status=\'closed\' )'
```
Comment (greezybacon) [inline include/class.ticket.php:1767]: So help me out. In the function `getStaffStats()` we're giving a staff member credit only if they currently own a closed ticket (ie. someone else could have actually closed it)? Should we transition to scanning the history table for staff stats?

Comment (protich) [inline include/class.ticket.php:1767]: These are not staff's historical stats - but live ticket counts based on staff's access.

Comment (greezybacon) [inline include/class.ticket.php:1767]: Closed is a realtime stat?

Comment (protich) [inline include/class.ticket.php:1767]: Basically the query gets count of closed tickets - in that context - then yes.

**Pull request merged**